packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -154,11 +155,22 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
       waitForDOMAndCameraDetails: {
         after: {
           0: {
-            target: 'detectFaceBeforeStart',
+            target: 'start',
             cond: 'hasDOMAndCameraDetails',
           },
-          // setting this to check every 500 ms sometimes caused detectFaceBeforeStart to be called twice
-          500: { target: 'waitForDOMAndCameraDetails' },
+          10: { target: 'waitForDOMAndCameraDetails' },
+        },
+      },
+      start: {
+        entry: ['drawStaticOval', 'initializeFaceDetector'],
+        always: [
```

Comment:
Once we reach the start state we wait for the user to click the `begin check` button. If the dev added the `skipStartScreen` flag then we can skip this

Reply:
Excluding the start screen is a UI concern and extraneously increases complexity in the state machine imo